src/objectExplorer/objectExplorerService.ts

```diff
@@ -87,7 +87,7 @@ export class ObjectExplorerService {
         const rootId = this._connectionManager.connectionStore.rootGroupId;
 
         if (!this._connectionGroupNodes.has(rootId)) {
-            this._logger.error(
+            this._logger.verbose(
```

Comment:
The issue here is less that the logging level is wrong and more that the OE isn't loaded when this is getting called.  Given that the repro is that the user has not opened OE but _is_ connecting, we should drop a breakpoint here and see what's triggering this, then decide whether we want to swallow this error conditionally (maybe if the map is empty, or some initalization flag isn't set) or code some alternate path to avoid calling this entirely.